mypy/constraints.py

```diff
@@ -18,8 +19,15 @@
 from mypy.argmap import ArgTypeExpander
 from mypy.typestate import TypeState
 
-SUBTYPE_OF: Final = 0
-SUPERTYPE_OF: Final = 1
+
+@enum.unique
+class SubType(enum.IntEnum):
+    SUBTYPE_OF = 0
+    SUPERTYPE_OF = 1
+
```

Comment:
If I recall correctly, mypyc currently only supports enum.Enum

Reply:
https://github.com/mypyc/mypyc/issues/721

Reply:
Thanks! I was not aware of that. 👍